packages/jest/src/setup.ts

```diff
@@ -7,21 +7,54 @@
 
 import { toBeAccessible } from './matcher';
 import { A11yConfig } from '@sa11y/preset-rules';
+import { AutoCheckOpts, registerSa11yAutomaticChecks } from './automatic';
+
+/**
+ * Options to be passed on to {@link setup}
+ */
+export type Sa11yOpts = {
+    autoCheckOpts: AutoCheckOpts;
+    // TODO (feat): add support for global opts to control formatting, filtering etc
+    // runOpts: RunOpts; // including ruleset, include/exclude selectors etc
+    // formatOpts: FormatOpts; // including format.Options etc
+    // filterOpts: FilterOpts; // including exception list filtering etc
+};
+
+/**
+ * Default options for automatic checks
+ */
+const defaultAutoCheckOpts: AutoCheckOpts = {
+    runAfterEach: false,
+    cleanupAfterEach: false,
+};
+
+const defaultSa11yOpts: Sa11yOpts = {
+    autoCheckOpts: defaultAutoCheckOpts,
+};
+
+/**
+ * Register Sa11y Jest API and automatic checks depending on {@link Sa11yOpts}
+ * @param opts - {@link Sa11yOpts} to opt-in to automatic checks
+ */
+export function setup(opts: Sa11yOpts = defaultSa11yOpts): void {
+    registerSa11yMatcher();
+    // Set defaults from env vars
+    const autoCheckOpts = opts.autoCheckOpts;
+    autoCheckOpts.runAfterEach = autoCheckOpts.runAfterEach || !!process.env.SA11Y_AUTO;
+    autoCheckOpts.cleanupAfterEach = autoCheckOpts.cleanupAfterEach || !!process.env.SA11Y_CLEANUP;
```

Comment:
Are the environment variables just for convenience, or do we have tools that require this?

Reply:
It is to turn on automatic checks without modifying the client setup code e.g. to setup a parallel build in a CI env.